--- a/scripts/selinux/genheaders/Makefile
+++ b/scripts/selinux/genheaders/Makefile
@@ -2,5 +2,7 @@
 hostprogs-y	:= genheaders
 HOST_EXTRACFLAGS += \
+	-I$(srctree)/arch/$(SRCARCH)/include/uapi \
+	-I$(objtree)/arch/$(SRCARCH)/include/generated/uapi \
 	-I$(srctree)/include/uapi -I$(srctree)/include \
 	-I$(srctree)/security/selinux/include
 
--- a/scripts/selinux/mdp/Makefile
+++ b/scripts/selinux/mdp/Makefile
@@ -2,5 +2,7 @@
 hostprogs-y	:= mdp
 HOST_EXTRACFLAGS += \
+	-I$(srctree)/arch/$(SRCARCH)/include/uapi \
+	-I$(objtree)/arch/$(SRCARCH)/include/generated/uapi \
 	-I$(srctree)/include/uapi -I$(srctree)/include \
 	-I$(srctree)/security/selinux/include
 
